components/raftstore/src/store/worker/read.rs

```diff
@@ -1153,6 +1196,26 @@ where
                             }
                         }
                     }
+                    RequestPolicy::ReadIndex => {
+                        TLS_LOCAL_READ_METRICS
+                            .with(|m| m.borrow_mut().local_received_follower_read_requests.inc());
```

Comment:
`local_received_follower_read_requests` should be recorded in `RequestPolicy::ReadIndexReplicaRead` branch.